chapters/data-analysis.tex

```diff
@@ -193,9 +227,13 @@ \subsection{Correction of data entry errors}
 create an automated workflow to identify, correct and document
 occurrences of duplicate entries. 
 
-Looking for duplicated entries is usually part of data quality monitoring,
-as is the only other reason to change the raw data during cleaning:
-correcting mistakes in data entry.
+As discussed in the previous chapter,
+looking for duplicated entries is usually part of data quality monitoring,
+and is typically addressed as part of that process.
+So, in practice, you will start writing data cleaning code during data collection.
+The other only other case when changes to the raw data are made during cleaning
+is also directly connected to data quality monitoring:
+it's when you need to correct mistakes in data entry.
```

Comment:
I do not agree. I think this should be in a correction file that is run immediately after the raw data is imported. We need to document changes made to the raw data. The corrected version of the raw data might be more correct, but it is not longer raw.

Reply:
Say data values or something, not `raw data` as it too associated with raw data as in the most original version of the data set

Reply:
done in 50e1d81